docs/content/reference/provider-infra/aws.md

```diff
@@ -0,0 +1,63 @@
+
+## AWS Components and Hypershift
+
+As you've seen, we've been showing all the components created, managed, not managed, etc... in the Hypershift life cycle. In this section we want to dissect who creates what and what not. It contains 4 stages:
```

Comment:
This intro makes no sense here. Let's just go with:
"In this section we want to dissect who creates what and what not. It contains 4 stages:"